index.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+require_once('src/controllers/add_comment.php');
+require_once('src/controllers/homepage.php');
+require_once('src/controllers/post.php');
+ 
+try {
+    if (isset($_GET['action']) && $_GET['action'] !== '') {
```

Comment:
Prefer yoda style in comparisons `if ('' !== $_GET['action'])`, `if ('post' === $_GET['action']`...